packaging/etc/tagger.yml

```
@@ -6,3 +6,33 @@ tagger:
         - tagger_managed_1
       absent:
         - tagger_absent_1
+  volumes:
+  - regex: '.+'
+    tags:
+      present:
+        - tagger_managed_1
+      absent:
+        - tagger_absent_1
+  nodebalancers:
+  - regex: 'ccm.+'
+    tags:
+      present:
+        - tagger_managed_1
+        - lke
+      absent:
+        - tagger_absent_1
+  domains:
+  - regex: '.+'
+    tags:
+      present:
+        - tagger_managed_1
+      absent: []
+  lkeclusters:
```

```suggestion
  lke_clusters:
```
Maybe?

This reads a bit odd to me. Open to opinions.

---

That's fine, I'm not biased either way. I was copying the API format of "LKEClusters" but it's more human readable with the underscore